### PR TITLE
Clean up more track flags

### PR DESCRIPTION
**Pull request by Gymnasiast**

### CODE REVIEW

File: test/testpaint/compat.c [hunk 1]
```diff
@@ -240,3 +240,8 @@ bool is_csg_loaded()
 {
     return false;
 }
+
+uint8 track_element_get_colour_scheme(const rct_map_element * mapElement)
+{
+    return mapElement->properties.track.colour & 0x3;
+}
```
Comment (janisozaur) [inline test/testpaint/compat.c]: tsk tsk

Comment (IntelOrca) [reply test/testpaint/compat.c]: @janisozaur I am surprised you haven't make CIs fail builds over this yet.

Comment (Gymnasiast) [reply test/testpaint/compat.c]: I don't understand what you two are on about. Isn't this the proper fix?

Comment (janisozaur) [reply test/testpaint/compat.c]: @Gymnasiast it's missing newline. It doesn't show the beautiful icon unless you look at the commit page directly.

screenshot-2017-10-4 clean up more track flags by gymnasiast pull request 6365 openrct2 openrct2

Comment (Gymnasiast) [reply test/testpaint/compat.c]: Oh, that? I already fixed it for track.c, but overlooked this one.

**Pull request merged**